File: src/main/java/com/example/estantevirtual/EstanteVirtualApplication.java
package com.example.estantevirtual;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cache.annotation.EnableCaching;

@SpringBootApplication
@EnableCaching
public class EstanteVirtualApplication {

    public static void main(String[] args) {
        SpringApplication.run(EstanteVirtualApplication.class, args);
    }

}

File: src/main/kotlin/com/estantevirtual/EstanteVirtualApplication.kt
package com.estantevirtual

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.cache.annotation.EnableCaching

@SpringBootApplication
@EnableCaching
class EstanteVirtualApplication

fun main(args: Array<String>) {
    SpringApplication.run(EstanteVirtualApplication::class.java, *args)
}
